rcl/src/rcl/arguments.c

```diff
@@ -139,11 +143,45 @@ rcl_arguments_get_param_overrides(
   return RCL_RET_OK;
 }
 
+rcl_ret_t
+rcl_arguments_get_log_levels(
+  const rcl_arguments_t * arguments,
+  rcl_log_levels_t ** log_levels)
+{
+  RCL_CHECK_ARGUMENT_FOR_NULL(arguments, RCL_RET_INVALID_ARGUMENT);
+  RCL_CHECK_ARGUMENT_FOR_NULL(arguments->impl, RCL_RET_INVALID_ARGUMENT);
+  RCL_CHECK_ARGUMENT_FOR_NULL(arguments->impl->log_levels, RCL_RET_INVALID_ARGUMENT);
+  RCL_CHECK_ARGUMENT_FOR_NULL(log_levels, RCL_RET_INVALID_ARGUMENT);
+
+  if (NULL != *log_levels) {
+    RCL_SET_ERROR_MSG("Output log level pointer is not null.");
+    return RCL_RET_INVALID_ARGUMENT;
+  }
+
+  *log_levels = rcl_log_levels_init(
```

Comment:
You shouldn't need to init before copying.

Reply:
I don't see there is a problem here. What's your suggestion?

Reply:
@ivanpauno 
misunderstood your meaning.
Updated.